### PR TITLE
Add gitlab error renderer

### PR DESCRIPTION
**Pull request by frankdekker**
Feature: add gitlab error renderer format

Added new renderer for gitlab format
Added new cli option --report. Options `checkstyle|gitlab`, defaults to checkstyle.
Added coverage

### CODE REVIEW

**Comment by frankdekker** [comment]
@daanheikens 
You're up, phpfci done :)

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.5/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd"
```
Comment (bram123) [inline phpunit.xml.dist:3]: Can you then also update composer.json to "^9.5" ?

**Review by bram123 — CHANGES_REQUESTED** [review]

File: phpunit.xml.dist (hunk 1 above)
Comment (frankdekker) [reply phpunit.xml.dist:3]: Odd, had removed 8.5, was default 9.4 then. Now changed to ^9.5

**Review by daanheikens — APPROVED** [review]
Looking good to me!

**Review by bram123 — APPROVED** [review]

**Pull request merged**